docs/egress.md

```diff
@@ -10,39 +10,94 @@ Accordingly, we have configured [a Caddy proxy](https://github.com/GSA-TTS/cg-eg
 
 - `uaa.fr.cloud.gov`: The [cloud.gov UAA server](https://cloud.gov/docs/management/leveraging-authentication/) which in turn uses GSA SecureAuth for authentication.
 - `google-analytics.com`: [DAP](https://digital.gov/guides/dap/), for web app analytics
```

Comment:
Let's remove `google-analytics`?

Reply:
I can try that change out in staging. I found a mention in Slack from April 2023 stating that we need it, but I don't know what that was based on.

I would like to make that change separately from these ones as a follow-up. It'd be convenient to make all the egress config changes together, but I don't want to add to the surface area of this update in case we do run into additional issues with the prod deploy.